components/core/tools/scripts/utils/build.py

```
@@ -0,0 +1,117 @@
+root_logger = logging.getLogger()
+root_logger.setLevel(logging.INFO)
+root_logger.addHandler(logging_console_handler)
+
+# Create logger
+logger = logging.getLogger(__name__)
+
+# Number of parent directories to reach project root from this script
+PROJECT_ROOT_DEPTH = 6
+
+def _run_subprocess(cmd: List[str], cwd: Optional[Path] = None) -> None:
+    logger.info(f"Running: {' '.join(cmd)} in {cwd if cwd else Path.cwd()}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)